src/test/scala/org/ergoplatform/modifiers/history/HeadersSpec.scala

```diff
@@ -25,24 +24,4 @@ class HeadersSpec extends ErgoPropertyTest {
     }
   }
 
-  property("Header.interlinks.tail should not contain genesis id") {
-    chain.headers.tail.foreach { r =>
-      r.interlinks.tail should not contain genesisId
-    }
-  }
-
-  property("Header Interlink(0) should always link to genesis block") {
-    chain.headers.tail.foreach { r =>
-      r.interlinks.head shouldEqual genesisId
-    }
-  }
-
-  property("Header interlink length should not be less than parents") {
```

Comment:
Looks like this test was not reproduced in PoPowAlgosSpec

Reply:
Not fixed yet?

Reply:
@catena2w , This check is done in `org.ergoplatform.modifiers.history.PoPowAlgosSpec$updateInterlinks`